critiquebrainz/frontend/templates/review/modify/base.html

```diff
@@ -86,32 +77,11 @@
       });
       if ('{{not review or review.is_draft}}'){
         $("#btn-draft").click(function(){
-          beingSaved = true;
-          $("#state").attr('value', 'draft');
-          $("#review-editor").submit();
+            beingSaved = true;
```

Comment:
Still some left :(

Reply:
Wait, I don't see it - it's 2 spaces, right?

Reply:
Nope, it's wrong :( I promise this kind of stuff won't happen in the future. My editor went mad on these.

Reply:
Haha, no issues